alot/commands/globals.py

```diff
@@ -553,6 +553,21 @@ def apply(self, ui):
             ui.buffer_open(buffers.TagListBuffer(ui, tags, self.filtfun))
 
 
+@registerCommand(MODE, 'namedqueries')
+class NamedQueriesCommand(Command):
+    """opens named queries buffer"""
+    def __init__(self, filtfun=lambda x: x, **kwargs):
```

Comment:
this doesn't match the definition in the docstring, I think we should either replace the lambda with `bool`, (which will return True for any string that != "").

Reply:
I don't quite see how a callable would match type `bool`, but ok..

Reply:
what I mean is, `lambda x: x` is  a function that takes a thing `T` and returns `T`, but the documentation says that the callable is a function that takes type `str` and returns type `bool`; `bool()` is a function that takes `T` and returns `bool`, a superset of what the documentation says. In most cases the lambda will work, since any type not bool that is used with a logical operator will have it's `__bool_()` method called, but there are cases where it wouldn't work, like if someone where to write `if filtfun(val) is True: ...`.